README.md

```diff
@@ -18,27 +18,15 @@ This repository contains Docker configuration aimed at Moodle developers and tes
 
 ## Quick start
 
-```bash
-# Set up path to Moodle code
-export MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
-# Choose a db server (Currently supported: pgsql, mariadb, mysql, mssql, oracle)
-export MOODLE_DOCKER_DB=pgsql
-
-# Ensure customized config.php for the Docker containers is in place
-cp config.docker-template.php $MOODLE_DOCKER_WWWROOT/config.php
```

Comment:
This still seems necessary for someone to quick start?

Reply:
Whoups your right. I added a `create-moodle-config` script and updated the readme in the latest commit.